sdk/package.json

```
@@ -25,7 +25,7 @@
 	"scripts": {
 		"build": "NODE_ENV=production tsup",
 		"dev": "NODE_ENV=development tsup",
-		"test": "vitest --watch=false"
+		"test": "vitest --config vitest.config.ts"
```

Default path so we can shorten this one
```suggestion
		"test": "vitest"
```